packages/jest-validate/README.md

```diff
@@ -0,0 +1,80 @@
+# jest-validate
+
+Provides handy validation errors and warnings for given configuration object.
+
+```
+npm install --save-dev jest-validate
+```
+
+## Usage
+
+```js
+import {validate} from 'jest-validate';
+
+validate(config: Object, validConfigSample: Object, deprecatedConfig: ?Object, options: ?Object);
+```
+
+## Customization
+
+By default `jest-validate` will print errors and warnings specific to Jest. You can however customize this behavior by providing `options` object as a last argument:
```

Comment:
We should aim to change the code so we can actually make this README generic and usable for others. I think react-native's packager could potentially benefit from this, too.

"jest-validate is a generic configuration validation tool that helps you with warnings and deprecation messages as well as showing users examples of correct configuration."

something like that :)